config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -602,7 +602,7 @@ presubmits:
               - '--node-test-args=--feature-gates=KubeletPSI=true --service-feature-gates=KubeletPSI=true --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/home/containerd/usr/local/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service"'
               - --node-tests=true
               - --provider=gce
-              - --test_args=--nodes=8 --focus="Summary API"
+              - --test_args=--nodes=8 --focus="Summary API|\[Feature:KubeletPSI\]"
```

Comment:
This feature is being removed in this PR: https://github.com/kubernetes/kubernetes/pull/134303

Why would this needs to be added here?

Also it is a beta FG enabled by default that doesn't need a special environment to run. All tests like this will execute as part of a regular NodeConformance tests execution